test/util/schemaform-utils.jsx

```diff
@@ -15,6 +17,7 @@ export class DefinitionTester extends React.Component {
     const uiSchema = props.uiSchema;
     const data = props.data;
     let schema = updateRequiredFields(props.schema, uiSchema, data, state);
+    schema = _.merge(schema, { definitions: props.definitions });
```

Comment:
Should be the other way around, I think: `_.merge({ definitions: props.definitions }, schema);`. That way the definitions in `schema` take precedence.